### PR TITLE
HW2 add link to AR Foundation Controllers repo

### PR DESCRIPTION
**Pull request by bieseah**

### CODE REVIEW

File: students/bs3243.md [hunk 1]
```diff
@@ -6,28 +6,10 @@ UNI: bs3243
...
-- Additional python time library that only deals with by the hour or minute, to simplify things. Could include a library for dealing with ranges of time since I have previously found working with restaurant opening times difficult
-- Javascript library that simplifies storing input values across page reloads
-- Python error simplifier. Catch statements typically have similar code, maybe there is a way to simplify this
-- Numpy data normalization library
-- Python code statistics library, simplify measuring speed and space while providing default, easy to read strings
-- Python to SQL library that simplifies queries
-- VS Code extension that allows users to see code snippets when hovering over areas of the page summary
-- Application for users to search for current exhibitions based off of artists. Most gallery searching websites/apps are organized by galleries, not artists. The only useful gallery app/tool is SeeSaw which is only available on Apple
+## Project Proposal
+- [ARFoundation-controllers](./projects/csharp/ARFoundation-controllers.md)
```
Comment (timkpaine) [inline students/bs3243.md:12]: update link path and re-squash 😈

File: students/bs3243.md [hunk 2]
```diff
@@ -6,28 +6,10 @@ UNI: bs3243
...
 
 ## Project Contributions
-
-- Blender
-- Numpy
-- Fabric.js
-- Node.js
-- Anime.js
-- jQuery
+- [transformers](./projects/python/transformers.md)
```
Comment (timkpaine) [inline students/bs3243.md:15]: update link path and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: students/bs3243.md (hunk 2 above)
Comment (bieseah) [reply students/bs3243.md]: done

File: students/bs3243.md (hunk 1 above)
Comment (bieseah) [reply students/bs3243.md]: done

**Pull request merged**